### PR TITLE
add `empty_option` to the form component

### PR DESCRIPTION
**Pull request by xylle**
Add empty option to select

### CODE REVIEW

**Comment by lovasoa** [comment]
Thank you ! You also need to document the new option, otherwise no one will know about it.

The documentation for the form component is here: https://github.com/sqlpage/SQLPage/blob/main/examples/official-site/sqlpage/migrations/01_documentation.sql#L232-L273

**Comment by lovasoa** [comment]
Ideally, you would also update the select example below, to demonstrate the usage of the new option.

**Comment by xylle** [comment]
Like that, it's good ?

File: examples/official-site/sqlpage/migrations/01_documentation.sql [hunk 1]
```diff
@@ -251,6 +251,7 @@ INSERT INTO parameter(component, name, description, type, top_level, optional) S
     ('max', 'The maximum value to accept for an input of type number', 'REAL', FALSE, TRUE),
     ('checked', 'Used only for checkboxes and radio buttons. Indicates whether the checkbox should appear as already checked.', 'BOOLEAN', FALSE, TRUE),
     ('multiple', 'Used only for select elements. Indicates that multiple elements can be selected simultaneously. When using multiple, you should add square brackets after the variable name: ''my_variable[]'' as name', 'BOOLEAN', FALSE, TRUE),
+    ('emptyoption', 'Used only for select elements. Add empty option at first', 'BOOLEAN', FALSE, TRUE),
```
Comment (lovasoa) [inline examples/official-site/sqlpage/migrations/01_documentation.sql:254]:
```suggestion
    ('empty_option', 'Only for inputs of type `select`. Adds an empty option before the ones defined in `options`.', 'BOOLEAN', FALSE, TRUE),
```

File: sqlpage/templates/form.handlebars [hunk 2]
```diff
@@ -75,6 +75,7 @@
                         {{~#if placeholder}} placeholder="{{placeholder}}" {{/if~}}
                         {{~#if create_new}} data-create_new={{create_new}} {{/if~}}
                         >
+                        {{~#if emptyoption}}<option value=""></option>{{/if~}}
```
Comment (lovasoa) [inline sqlpage/templates/form.handlebars:78]:
```suggestion
                        {{~#if empty_option}}<option value=""></option>{{/if~}}
```

**Review by lovasoa — CHANGES_REQUESTED** [review]
I suggest we name it `empty_option` to follow the snake_case wording of other sqlpage component properties.

**Pull request merged**